spec/spec_helper.rb

```diff
@@ -111,12 +111,6 @@
       full_context = description.chomp(test).strip
       name = full_context.split(/\s+/, 2).join('/').underscore.gsub(/\./,'/').gsub(/[^\w\/]+/, '_').gsub(/\/$/, '')
 
-      #disable re-recording Azure cassettes until tests are removed:
```

Comment:
This is no longer necessary, as the `re_record_interval` for all specs is now `nil`.